docker-compose.yml

```
@@ -0,0 +1,28 @@
+version: "3.7"
+services:
+  minio:
+    image: minio/minio
+    command: server /data
+    healthcheck:
+      test: ["CMD", "curl", "http://localhost:9000"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    ports:
+      - target: 9000
+        published: 9000
+        protocol: tcp
+        mode: host
```

```suggestion
  minio:
    image: minio/minio
    command: server /data
    healthcheck:
      test: ["CMD", "curl", "http://localhost:9000"]
      interval: 10s
      timeout: 5s
      retries: 5
    ports:
      - target: 9000
        published: 9000
        protocol: tcp
        mode: host
    volumes:
      - ./data:/data
```
Do we want to have some basic persistence here?

---

I hadn't really considered that. I am mostly using this for testing, so right now I don't really need persistence, but ya I can see where this would be useful.

---

I'm going to leave it off this pr for now, but definitely something to keep in mind!